gui/tasks/forms.py

```diff
@@ -186,6 +187,8 @@ def check_rpath_exists(self):
         exists or not. Returns TRUE rpath is a directory
         and exists, else FALSE"""
 
+        self.ssh_host_keyscan()
```

Comment:
I agree with @williambr about only saving the key if the user explicitly says to do so. How about this, we continue to copy the replication gui and add a button in there that does this, instead of doing this on save and all? That way we can satisfy the strict host checking in the rpath validation as well as satisfy the key not being saved without the user's intent to do so.